aws/resource_aws_dx_hosted_private_virtual_interface_accepter.go

```diff
@@ -0,0 +1,160 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/arn"
+	"github.com/aws/aws-sdk-go/service/directconnect"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsDxHostedPrivateVirtualInterfaceAccepter() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsDxHostedPrivateVirtualInterfaceAccepterCreate,
+		Read:   resourceAwsDxHostedPrivateVirtualInterfaceAccepterRead,
+		Update: resourceAwsDxHostedPrivateVirtualInterfaceAccepterUpdate,
+		Delete: resourceAwsDxHostedPrivateVirtualInterfaceAccepterDelete,
+		Importer: &schema.ResourceImporter{
+			State: resourceAwsDxHostedPrivateVirtualInterfaceAccepterImport,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"arn": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"virtual_interface_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"vpn_gateway_id": {
+				Type:          schema.TypeString,
+				Optional:      true,
+				ForceNew:      true,
+				ConflictsWith: []string{"dx_gateway_id"},
+			},
+			"dx_gateway_id": {
+				Type:          schema.TypeString,
+				Optional:      true,
+				ForceNew:      true,
+				ConflictsWith: []string{"vpn_gateway_id"},
+			},
+			"tags": tagsSchema(),
```

Comment:
Unless the "confirmation" itself has some sort of resource tagging, this should be removed 👍

Reply:
The accepting side of the interface IS allowed to tag the VIF (this is done in `dxVirtualInterfaceUpdate`).

Reply:
The tags can only be set on the accepting side - The allocating side can't see the tags (I guess they live in the owning account).